R/compat-purrr.R

```diff
@@ -0,0 +1,13 @@
+imap <- function(.x, .f, ...) {
```

Comment:
I'd just dump these in `utils.R` since we're starting to standardise our compat files (see e.g. `usethis::use_tidy_dependencies`)

Reply:
Sounds great to me to standardise the compat files! :-)